fbgemm_gpu/experimental/gen_ai/src/quantize/ck_extensions/kernels/fp8_rowwise_common.h

```diff
@@ -201,4 +199,3 @@ at::Tensor f8f8bf16_rowwise_impl(
   return Y;
 }
 
```

Comment:
@acoskunses-AMD can you remove the blank lines at the end of each file? Otherwise we cannot land it internally due to some lint check